benchmarks/benchmarks/stats.py

```diff
@@ -19,4 +18,84 @@ def setup(self, *args):
     def time_anderson_ksamp(self):
         with warnings.catch_warnings():
             warnings.simplefilter('ignore', UserWarning)
-            anderson_ksamp(self.rand)
+            stats.anderson_ksamp(self.rand)
+
+
+class CorrelationFunctions(Benchmark):
+    param_names = ['alternative']
+    params = [
+        ['two-sided', 'less', 'greater']
+    ]
+
+    def setup(self, mode):
+        a = np.random.rand(2,2)*10
```

Comment:
That's related to the pep8 error codes that are excluded in ``tox.ini`` in the root dir of the repo. If you copy this file to some temporary directory outside the source tree and directly run ``pep8`` on it, you'll get more violations.